webapp/src/js/directives/messages-list.js

```diff
@@ -18,6 +18,6 @@ angular.module('inboxDirectives').component('mmMessagesList', {
   controllerAs: 'messagesListCtrl',
   bindings: {
     loading: '<',
-    messages: '<'
+    conversations: '<'
```

Comment:
I refactored this for clarity: "conversations" are groups of "messages" to/from a contact. Having both called "messages" was crazy...